apollo-router/src/configuration/mod.rs

```diff
@@ -1255,3 +1265,16 @@ fn default_graphql_path() -> String {
 fn default_graphql_introspection() -> bool {
     false
 }
+
+/// Configuration for Batching
+#[derive(Debug, Default, Clone, Deserialize, Serialize, JsonSchema)]
+#[serde(deny_unknown_fields)]
+#[serde(default)]
```

Comment:
```suggestion
#[serde(deny_unknown_fields, default)]
```

Reply:
This has changed with the change to the spec. There is no default on the struct now.